uniref-rest/src/test/resources/application.properties

```diff
@@ -55,7 +55,8 @@ async.download.retryMaxCount=3
 async.download.retryQueueName=retry_uniref_stream_requests_q
 async.download.rejectedQueueName=rejected_uniref_stream_requests_q
 async.download.heartbeat.enabled=true
-async.download.heartbeat.interval=2
+async.download.heartbeat.results-interval=2
+async.download.heartbeat.ids-interval=2
 
```

Comment:
Lets use resultsInterval and idsInterval instead with dash. It will be consistent and easy to search.